docs/optimization_guide/nncf/ptq/quantization_w_accuracy_control.md

```diff
@@ -51,14 +51,33 @@ Now, you can run quantization with accuracy control. The following code snippet
       :fragment: [quantization]
 
 
-``max_drop`` defines the accuracy drop threshold. The quantization process stops when the degradation of accuracy metric on the validation dataset is less than the ``max_drop``.
+* ``max_drop`` defines the accuracy drop threshold. The quantization process stops when the degradation of accuracy metric on the validation dataset is less than the ``max_drop``. The default value is 0.01.
```

Comment:
What happens if you set max_drop to an unreasonably low value that cannot be reached?

Reply:
Just like POT it will stop the quantization and report an error that the max_drop value can't be reached. I've clarified it in the text.